sdk/python/setup.py

```diff
@@ -24,13 +24,14 @@ def run(self):
                 raise
 
 def readme():
-    with open('README.rst') as f:
+    with open('README.md', encoding='utf-8') as f:
         return f.read()
 
 setup(name='pulumi_random',
       version='${VERSION}',
       description='A Pulumi package to safely use randomness in Pulumi programs.',
       long_description=readme(),
+      long_description_content_type='text/markdown',
```

Comment:
what's up with this file?

Reply:
This is a consequence of updating to a newer version of the bridge, which replaces the old rst docs for Python with Markdown.

Reply:
Gotcha.  thanks!